### PR TITLE
Bugfix - TRSV2-385 - Invalid phone number length

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## Context

- ensure delivery type is switched to `email` from `sms` when saved phone number is invalid, pertaining to the login journey

### CODE REVIEW

**Review by osimuka — APPROVED** [review]

File: trade_remedies_api/core/utils.py [hunk 1]
```diff
@@ -96,6 +100,9 @@ def convert_to_e164(raw_phone, country=None):
         e164_phone = phonenumbers.format_number(
             phone_representation, phonenumbers.PhoneNumberFormat.E164
         )
+        if country == "GB" and len(e164_phone) != 13:
```
Comment (chris-pettinga) [inline trade_remedies_api/core/utils.py:103]: What if it's not a UK number, is there a phonenumbers is_valid() method we can use?

**Review by chris-pettinga — APPROVED** [review]
Looks good thanks Yusuf well done. Just left a small comment

File: trade_remedies_api/core/utils.py [hunk 2]
```diff
@@ -76,7 +76,11 @@ def rekey(item, key, rekey_as=None):
     return rekeyed
 
 
-def convert_to_e164(raw_phone, country=None):
+class InvalidPhoneNumberFormatException(Exception):
```
Comment (chris-pettinga) [inline trade_remedies_api/core/utils.py:79]: I'd rather this be in a separate file called exceptions.py just to keep the code clean

File: trade_remedies_api/core/utils.py (hunk 1 above)
Comment (yusufsheiqh) [reply trade_remedies_api/core/utils.py:103]: Yes, there is: `is_possible_number(z)`... and that seems like a much better option. Will add a test or two to capture this.

**Review by nahmed99 — APPROVED** [review]

File: trade_remedies_api/core/utils.py (hunk 1 above)
Comment (yusufsheiqh) [reply trade_remedies_api/core/utils.py]: Updated in https://github.com/uktrade/trade-remedies-api/pull/304/commits/20263bdebb5f91297a7646344f82c38ce4b664e4.

**Review by osimuka — APPROVED** [review]

**Pull request merged**